### ISSUE TITLE
Add support for latest syncstorage-rs version

### ISSUE THREAD

**Issue by jackjackk** [jeena/fxsync-docker#3]
First of all, thanks a lot for this repo, it was **the only way** I was ever able to switch from the Python-2 syncstorage to the more recent Rust version.

Now, it seems that some breaking changes from 0.13.17 to 0.14 prevent docker setups that used to work from working (e.g. see https://github.com/mozilla-services/syncstorage-rs/issues/1482). From trying to follow the latest issues, it seems some google cloud dependencies have been added (which also defeat the privacy purposes of self-hosting). Ultimately, also the docker setup suggested in this repo, with version 0.14+ as well as 0.15.1 (currently the latest), does not work, at least according to my basic attempts.

I cannot solve this issue on my own, but I am happy to help and do whatever I can to keep this essential repo up-to-date!

**Comment by jeena** [comment]
Yeah, it seems we will need to wait until it's possible to do it with the upstream code. I don't think we will be able to fix it with the docker setup here.

My guess is that we need to vote on the issue you linked to, so that they understand that there are many of us with that issue.

**Comment by zilexa** [comment]
Any news on this?
Also, why not simply use 1 mariadb container. This is mostly for home users.. no need to split each and every task in a container.

**Comment by jackjackk** [comment]
@zilexa I am still running the old version, and haven't had the time to look into this, but the original issue I refer to (https://github.com/mozilla-services/syncstorage-rs/issues/1482) now mentions a repo (https://github.com/porelli/firefox-sync) that seems to host an up-to-date self-hostable docker version. Haven't tried myself though.

**Comment by jeena** [comment]
I'm also still running the old version which is working good enough for me for now, but yeah it's not a long term sollution. The https://github.com/porelli/firefox-sync looks very promising, and I don't think it's necessary to move the database or anything, just replace this setup with the new one and start syncing to the new one instead, then Firefox will move everything from local to that new one.
